linker-interface/shared/src/test/scala/org/scalajs/linker/interface/ModuleInitializerFingerprintTest.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.interface
+
+import org.junit.Test
+import org.junit.Assert._
+
+class ModuleInitializerFingerprintTest {
+
+  lazy val moduleInitializers: Stream[ModuleInitializer] = {
+    lazy val genMainMethod = for {
+      className <- Stream("Test1", "Test2")
+      methodName <- Stream("main1", "main2")
+    } yield ModuleInitializer.mainMethod(className, methodName)
+
+    lazy val genMainMethodWithArgs = for {
+      className <- Stream("Test1", "Test2")
+      methodName <- Stream("main1", "main2")
+      args <- Stream(List(), List("arg1"), List("arg1", "arg2"))
+    } yield ModuleInitializer.mainMethodWithArgs(className, methodName, args)
+
+    genMainMethod ++ genMainMethodWithArgs
+  }
+
+  @Test
+  def noFingerprintCollision(): Unit = {
+    for ((a, b) <- moduleInitializers.zip(moduleInitializers.tail)) {
+      assertNotEquals(ModuleInitializer.fingerprint(a), ModuleInitializer.fingerprint(b))
+    }
+  }
+}
```

Comment:
Consider simplifying this test as well for consistency.

Reply:
Fixed in 2749f3ee1fd7cf9124217651c7a26130739fcd1d